### PR TITLE
WIP: Deployment attempt to AWS

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -35,5 +35,31 @@
...
+                    ihp.nixosModules.appWithPostgres
+                    ({ ... }: {
+                        security.acme.defaults.email = "no-reply@tpp-qa.gizra.site";
+
+                        services.ihp = {
+                            domain = "tpp-qa.gizra.site";
+                            migrations = ./Application/Migration;
+                            schema = ./Application/Schema.sql;
+                            fixtures = ./Application/Fixtures.sql;
+                            sessionSecret = "M$cmzMCEx7xfL-5_q6%9cpve_0BAd5BbDaOtzhv7";
```
Comment (mpscholten) [inline flake.nix:53]: The session secret should not be public, anyone with knowledge of the `sessionSecret` can write to the session cookie (e.g. override the logged in user or something else)

Comment (AronNovak) [reply flake.nix:53]: That's right, this is a dummy PR that we will throw away, we do not intend to deploy this project. Also we won't use this secret elsewhere.

Comment (amitaibu) [reply flake.nix:53]: @mpscholten Yeah, we've done it so you'd have access to the code 😄 
We can also invite you to the dummy AWS instance - so you'd have full visibility of the problem.

**Comment by amitaibu** [comment]
Closed in favor of https://github.com/amitaibu/ihp-landing-page/pull/34

**Pull request closed without merging**